cpython/Lib/ctypes/__init__.py

```diff
@@ -109,6 +109,18 @@ class CFunctionType(_CFuncPtr):
     if _os.name == "ce":
         # 'ce' doesn't have the stdcall calling convention
         _FUNCFLAG_STDCALL = _FUNCFLAG_CDECL
+    if _os.name == "uwp_os":
+        # LoadLibrary in 'uwp_os' really calls LoadPackagedLibrary which requires
+        # a relative path to the app install directory.  
+        def make_dlopen_wrapper():
```

Comment:
I needed the make_dlopen_wrapper function or else it won't be able to capture the original value of "_dlopen".